src/main/scala/org/themaniacs/core/ManiacCore.scala

```diff
@@ -19,19 +20,26 @@ object ManiacCore {
 
   var logger: Option[log4j.Logger] = None
 
+
+  @SidedProxy(clientSide = "org.themaniacs.core.proxy.ClientProxy", serverSide = "org.themaniacs.core.proxy.ServerProxy")
+  var proxy: CommonProxy = null
```

Comment:
does this have to be public? might be more beneficial to use i private val here, combined with a public getter